src/Auth/Adapter/AuthenticationAdapter.php

```diff
@@ -422,9 +423,10 @@ protected function notifySystemAdmin($userData)
 
         // Raise a new notification event
         $event = new NotificationEvent('User Admin', 'Login - Failed');
+        $personName = Format::name('', $this->session->get('preferredName'), $this->session->get('surname'), 'Staff', false, true);
 
         $event->addRecipient($this->session->get('organisationAdministrator'));
-        $event->setNotificationText(sprintf(__('Someone failed to login to account "%1$s" 3 times in a row.'), $userData['username']));
+        $event->setNotificationText(__('{person} failed to login to account {username} 3 times in a row.', ['person' => $personName, 'username' => $userData['username']]));
```

Comment:
Hi Ali, although at first glance it makes sense to refactor this message, unfortunately since the user hasn't successfully logged in at this point, there is no name information to draw on. In this edge case, I think we need to leave the word "someone" in this message. Can you revert your changes to this file? Thanks.